src/components/disconnectWallet.tsx

```diff
@@ -0,0 +1,20 @@
+import { Button, Tooltip } from '@chakra-ui/react';
+import React from 'react'
+import useWalletAccount from '../utils/persistAccount';
+import { BsFillEjectFill } from "react-icons/bs";
+
+const DisconnectWalletButton = () => {
+  const [account, setAccount] = useWalletAccount();
+
+  const handleDisconnect = () => {
+    setAccount();
+    localStorage.removeItem('walletAccount');
+  }   
+  return (
+    <Tooltip label='Disconnect Wallet'>
+      <Button onClick={handleDisconnect}><BsFillEjectFill /> </Button>
+    </Tooltip>
+  );
+}
+
+export default DisconnectWalletButton;
```

Comment:
this is the 'eject button' 
![image](https://user-images.githubusercontent.com/95011498/156108227-b53091f3-787d-4563-9e43-9725479341a6.png)

it should only show if an account is connected and if you click it "disconnects" the account. I put "disconnect" in quotes because all it does is clear our localStorage account info.. apparently that's the only way to disconnect these wallets according to https://ethereum.stackexchange.com/questions/83914/how-to-disconnect-metamask-wallet-using-web3modal